samples/apply-style-to-wms-layer/README.metadata.json

```
@@ -0,0 +1,31 @@
+{
+    "category": "Layers",
```

According to design:
```suggestion
    "category": "Visualization",
```

---

Swift has it as `Layers`
https://github.com/Esri/arcgis-maps-sdk-swift-samples/blob/v.next/Shared/Samples/Apply%20style%20to%20WMS%20layer/README.metadata.json#L2
I'll follow the design